src/BaGet/Program.cs

```diff
@@ -38,29 +40,27 @@ public static async Task Main(string[] args)
             {
                 var host = CreateWebHostBuilder(args).Build();
 
+                // Todo: Don't know how to get the options without rebuilding the host!
```

Comment:
I don't know how to get the configuration here in another way...

Reply:
I would follow this guide to capture configurations at startup: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/web-host?view=aspnetcore-3.1#override-configuration

I would grab the `IConfigurationRoot`, look for a `Urls` config. If it exists, call [`UseUrls`](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-3.1#urls).